Makefile

```diff
@@ -22,8 +22,8 @@ ALL_SRC := $(shell find . -name '*.go' \
 # ALL_PKGS is used with 'go cover' and 'golint'
 ALL_PKGS := $(shell go list $(sort $(dir $(ALL_SRC))))
 UNAME := $(shell uname -m)
-#Race flag is not supported on s390x architecture
-ifeq ($(UNAME), s390x)
+#Race flag is not supported on s390x/ppc64le architecture
+ifeq ($(UNAME),$(filter $(UNAME), s390x ppc64le))
```

Comment:
could we DRY this more?  Perhaps define a function

Reply:
Are you looking for a Makefile function for the RACE variable assignment?

Reply:
A function that will encapsulate the fact that we're checking two `uname`s instead of one, so that if we add a 3rd one in the future there'd be just one place to change in the Makefile.

Reply:
I'm not a Makefile expert, if you can point me to an example I will review it and apply the changes as required.

Reply:
I'm not an expert either. Here's example of functions: https://coderwall.com/p/cezf6g/define-your-own-function-in-a-makefile

At minimum, we could consider moving the list of "special" architectures into a variable so that, again, only one place needs to change.

Reply:
actually -- and sorry for not catching this sooner, the race flag has been supported for Power for a few releases now. I've asked someone else to carry this PR and will have her link to this one when she submits. Thanks @yurishkuro for the review.